src/sidebar/components/test/annotation-publish-control-test.js

```diff
@@ -70,6 +70,12 @@ describe('AnnotationPublishControl', () => {
     $imports.$restore();
   });
 
+  it('should not render if group is missing', () => {
+    fakeStore.getGroup.returns(undefined);
+    const wrapper = createAnnotationPublishControl();
+    assert.isFalse(wrapper.find('annotation-publish-control').exists());
```

Comment:
```suggestion
    assert.isFalse(wrapper.find('.annotation-publish-control').exists());
```

The CSS selector used here is an element selector, whereas you meant to match the `annotation-publish-control` _class_.

Reply:
this is a total face palm